bank2ynab.conf

```diff
@@ -30,38 +33,165 @@ Plugin =
+Input Columns = skip,Date,skip,Memo,skip,skip,skip,skip,skip,skip,skip,Payee,skip,skip,Inflow,skip,skip
+
+[DK Nordea]
+# Issue #92
+Source CSV Delimiter = ;
+# Bogført;Tekst;Rentedato;Beløb;Saldo
+# Reserveret;Nordea Pay Lebara; ;-100,00;999,99
+# 28-09-2016;Nordea pay køb Foreningen StudenterDen 27.09 kl. 20.03;28-09-2016;-54,00;999,99
+Input Columns = Date,Payee,skip,Inflow,skip
 
 [IE AIB Ireland]
-Source Filename Pattern = Transaction_Export_
-# Posted Account     Posted Transactions Date    Description     Debit Amount    Credit Amount  Balance Transaction Type
+# survey response #16
+Use Regex For Filename = True
+Source Filename Pattern = Transaction_Export_[0-9]{2}.[0-9]{2}.[0-9]{4}_[0-9]{2}.[0-9]{2}
+# Posted Account,Posted Transactions,Date,Description,Debit Amount,Credit Amount,Balance,Transaction Type
 Input Columns = skip,Date,Payee,Inflow,Outflow,skip,skip
 Use Payees for Memo = True
 
 [IE Bank of Ireland]
 Source Filename Pattern = TransactionExport
 Input Columns = Date,Payee,Outflow,Inflow,skip
 
-[UK Co-operative Bank]
-Input Columns = Date,Payee,Memo,Inflow,Outflow,skip
+[MV Bank of Maldives, checking]
+# survey response #18
+Source Filename Pattern = statement
+# "Posted Date","Effective Date","Transaction Code Description","Reference Number","FT Reference Number","Transaction Description 1","Transaction Description 2","Transaction Description 3","Debit Amount","Credit Amount","Running Balance"
+Input Columns = Date,skip,skip,skip,skip,Memo,skip,skip,Inflow,Outflow,skip
+
+[NL ING]
+Use Regex for Filename = True
+Source Filename Pattern = NL[0-9]{2}INGB[0-9]{9}_[0-9]{2}_[0-9]{2}_[0-9]{4}_[0-9]{2}_[0-9]{2}_[0-9]{4}.csv
+Header Rows = 1
+Footer Rows = 0
+Input Columns = Date,Payee,skip,skip,skip,CDFlag,Amount,skip,Memo
+Date Format = YYYYMMDD
+Inflow or Outflow Indicator = 5,Af,Bij
+
+[NL Rabobank]
+# survey response #14
+Source Filename Pattern = transactions
+Source Filename Extension = .txt
+Source Has Column Headers = False
+Input Columns = skip,skip,Date,CDFlag,Inflow,skip,Payee,skip,skip,skip,Memo,skip
+Date Format = %%Y%%m%%d
+Inflow or Outflow Indicator = 3,C,D
+
+[NO DNB]
+Use Regex For Filename = True
+Source Filename Pattern = maanedtransaksjonliste-[a-z]{3}
+Source CSV Delimiter = ;
+Input Columns = Date,Payee,skip,Outflow,Inflow
+
+[Personal Capital]
+# Issue #144
+Use Regex For Filename = True
+Source Filename Pattern = [A-Z][a-z]{2} [0-9]{2}-[0-9]{4} thru [A-Z][a-z]{2} [0-9]{2}-[0-9]{4} transactions
 Source CSV Delimiter = ,
+Source Has Column Headers = True
+Input Columns = Date,Payee,Memo,skip,skip,Inflow
+
+[SE Handelsbanken]
+Source Filename Pattern = kontotransactionlist
+Source Filename Extension = .xls
+Input Columns = skip,Date,Payee,Inflow,skip
+Source CSV Delimiter = ;
+Header Rows = 7
+Plugin = handelsbanken
+# don't delete source, so we can debug test
+Delete Source File = False
+
```

Comment:
Remove non-deletion

Reply:
You're saying these lines should be removed, right? I'll do it now.